Assets/GothicVR/Scripts/Manager/WayNetHelper.cs

```diff
@@ -53,6 +53,15 @@ public static WayPoint FindNearestWayPoint(Vector3 lookupPosition)
             return nearestWayPoint.Value;
         }
 
+        public static WayPoint FindSecondNearestWayPoint(Vector3 lookupPosition)
```

Comment:
Really? Don't tell me Daedalus has a function for that? Sounds stupid to me...
Maybe you can merge it with FindNearestWayPoint and add this one line as bool parameter?